diff --git a/node_modules/@capacitor/cli/dist/config.js b/node_modules/@capacitor/cli/dist/config.js
index ce072ec..9f9abbe 100644
--- a/node_modules/@capacitor/cli/dist/config.js
+++ b/node_modules/@capacitor/cli/dist/config.js
@@ -231,7 +231,7 @@ async function loadIOSConfig(rootDir, extConfig) {
     const nativeProjectDirAbs = (0, path_1.resolve)(platformDirAbs, nativeProjectDir);
     const nativeTargetDir = `${nativeProjectDir}/App`;
     const nativeTargetDirAbs = (0, path_1.resolve)(platformDirAbs, nativeTargetDir);
-    const nativeXcodeProjDir = `${nativeProjectDir}/App.xcodeproj`;
+    const nativeXcodeProjDir = `${nativeProjectDir}/Fleets.xcodeproj`;
     const nativeXcodeProjDirAbs = (0, path_1.resolve)(platformDirAbs, nativeXcodeProjDir);
     const nativeXcodeWorkspaceDirAbs = (0, promise_1.lazy)(() => determineXcodeWorkspaceDirAbs(nativeProjectDirAbs));
     const podPath = (0, promise_1.lazy)(() => determineGemfileOrCocoapodPath(rootDir, platformDirAbs, nativeProjectDirAbs));
@@ -279,7 +279,7 @@ function determineOS(os) {
     return "unknown" /* OS.Unknown */;
 }
 async function determineXcodeWorkspaceDirAbs(nativeProjectDirAbs) {
-    return (0, path_1.resolve)(nativeProjectDirAbs, 'App.xcworkspace');
+    return (0, path_1.resolve)(nativeProjectDirAbs, 'Fleets.xcworkspace');
 }
 async function determineIOSWebDirAbs(nativeProjectDirAbs, nativeTargetDirAbs, nativeXcodeProjDirAbs) {
     const re = /path\s=\spublic[\s\S]+?sourceTree\s=\s([^;]+)/;
